kolibri/plugins/coach/assets/src/constants/index.js

```diff
@@ -16,6 +16,9 @@ export const PageNames = {
   QUIZ_SECTION_EDITOR: 'QUIZ_SECTION_EDITOR',
   QUIZ_REPLACE_QUESTIONS: 'QUIZ_REPLACE_QUESTIONS',
   QUIZ_SELECT_RESOURCES: 'QUIZ_SELECT_RESOURCES',
+  SELECT_FROM_RESOURCE: 'SELECT_FROM_RESOURCE',
+  BOOK_MARKED_RESOURCES: 'BOOK_MARKED_RESOURCES',
```

Comment:
Of these three, I am only seeing this new PageName being used in the routes below?